src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String DATA_FIELD_SEPARATOR = ",";
```

Comment:
Consider using a more descriptive constant name instead of `DATA_FIELD_SEPARATOR`. For example, `CSV_FIELD_SEPARATOR`. 🤔